receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -37,16 +37,18 @@ func newRemoteWriteReceiver(settings receiver.Settings, cfg *Config, nextConsume
 		server: &http.Server{
 			ReadTimeout: 60 * time.Second,
 		},
+		interRequestCache: make(map[uint64]pmetric.ResourceMetrics),
 	}, nil
 }
 
 type prometheusRemoteWriteReceiver struct {
 	settings     receiver.Settings
 	nextConsumer consumer.Metrics
 
-	config *Config
-	server *http.Server
-	wg     sync.WaitGroup
+	config            *Config
+	server            *http.Server
+	wg                sync.WaitGroup
+	interRequestCache map[uint64]pmetric.ResourceMetrics
```

Comment:
```suggestion
	rmCache map[uint64]pmetric.ResourceMetrics
```

In the past we thought we would need two caches, and that's why we had intra and inter request cache. Now that we only have one, maybe naming them intra/inter doesn't make sense anymore. What do you think about renaming to something simpler? :)